### PR TITLE
Create user view now looks like other create views.

### PR DESCRIPTION
**Pull request by eristow**

### CODE REVIEW

File: modules/users/client/views/admin/create-user.client.view.html [hunk 1]
```diff
@@ -46,12 +48,13 @@ <h3 class="col-md-12 text-center">Sign up using your email</h3>
           <label>Password Requirements</label>
           <progressbar value="requirementsProgress" type="{{requirementsColor}}"><span style="color:white; white-space:nowrap;">{{requirementsProgress}}%</span></progressbar>
         </div>
-        <div class="text-center form-group">
+        <div class="form-group pull-right">
           <button type="submit" class="btn btn-primary">Sign up</button>
+          <input type="button" value="Cancel" class="btn" ui-sref="admin.users"/>
           <!--&nbsp; or&nbsp;
           <a ui-sref="authentication.signin" class="show-signup">Sign in</a-->
```
Comment (dkwhite200) [inline modules/users/client/views/admin/create-user.client.view.html]: Does this comment need to be here?

**Review by dkwhite200 — COMMENTED** [review]
Hey there is a commented out portion of code. Does it need to be left in?

File: modules/users/client/views/admin/create-user.client.view.html (hunk 1 above)
Comment (eristow) [reply modules/users/client/views/admin/create-user.client.view.html]: That's an artifact from the original sign up page. I don't think we'll need it, so I'll remove it. Not gonna push up the changes until I get some more reviews in. I don't want a commit for each review.

**Review by wcampbell09 — DISMISSED** [review]
Good stuff

**Review by wcampbell09 — APPROVED** [review]
nice

**Review by dkwhite200 — APPROVED** [review]

**Pull request merged**